From 4623b2aab891fdbeda0486ad00f1fa2a7f5533fe Mon Sep 17 00:00:00 2001
From: "Richard W.M. Jones" <rjones@redhat.com>
Date: Thu, 19 Jan 2023 08:51:30 +0000
Subject: [PATCH] -o kubevirt: Fix mistake in error message

The error message was copied from output_local.ml:

virt-v2v: error: -o local: -op option cannot be used in this output mode

Reported-by: Xiaodai Wang
(cherry picked from commit be322ed49b1131e30a0b70f16d31fd29fca32b81)
---
 output/output_kubevirt.ml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/output/output_kubevirt.ml b/output/output_kubevirt.ml
index 63ee8197..0a74dbbe 100644
--- a/output/output_kubevirt.ml
+++ b/output/output_kubevirt.ml
@@ -47,7 +47,7 @@ module Kubevirt = struct
     if options.output_options <> [] then
       error (f_"no -oo (output options) are allowed here");
     if options.output_password <> None then
-      error_option_cannot_be_used_in_output_mode "local" "-op";
+      error_option_cannot_be_used_in_output_mode "kubevirt" "-op";
 
     (* -os must be set to a directory. *)
     let output_storage =
